avm/ptn/lz/sub-vending/modules/subResourceWrapper.bicep

```
@@ -839,7 +856,23 @@ module createLzNsg 'br/public:avm/res/network/network-security-group:0.5.1' = [
   }
 ]
 
-module createLzRouteTable 'br/public:avm/res/network/route-table:0.4.1' = [
+module createLzNsgStandalone 'br/public:avm/res/network/network-security-group:0.5.1' = [
+  for (nsg, i) in networkSecurityGroups: if (!empty(networkSecurityGroups) && !empty(networkSecurityGroupResourceGroupName)) {
+    scope: resourceGroup(subscriptionId, virtualNetworkResourceGroupName)
```

```suggestion
    scope: resourceGroup(subscriptionId, networkSecurityGroupResourceGroupName)
```

shouldnt this be this?